config/routes.rb

```diff
@@ -1,5 +1,11 @@
 Rails.application.routes.draw do
   root 'homes#index'
   devise_for :users
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  get '/podcasts', to: "static_pages#index"
```

Comment:
we should use either the `homes` or `static_pages` controller, no need to have both